tests/accuracy_utils.py

```diff
@@ -26,37 +26,83 @@
 sizes_pow_2 = [2**d for d in range(4, 11, 2)]
 sizes_noalign = [d + 17 for d in sizes_pow_2]
 sizes_1d = sizes_one + sizes_pow_2 + sizes_noalign
-sizes_2d_nc = [1, 16, 64, 1000]
-sizes_2d_nr = [1, 5, 1024]
+sizes_2d_nc = [1] if TO_CPU else [1, 16, 64, 1000]
+sizes_2d_nr = [1] if TO_CPU else [1, 5, 1024]
 
 UT_SHAPES_1D = list((n,) for n in sizes_1d)
 UT_SHAPES_2D = list(itertools.product(sizes_2d_nr, sizes_2d_nc))
-POINTWISE_SHAPES = [(1024, 1024), (16, 1024, 256), (16, 128, 64, 64), (20, 320, 15)]
+POINTWISE_SHAPES = (
+    [(2, 19, 7)]
+    if TO_CPU
+    else [(1,), (1024, 1024), (20, 320, 15), (16, 128, 64, 60), (16, 7, 57, 32, 29)]
+)
+SPECIAL_SHAPES = (
+    [(2, 19, 7)]
+    if TO_CPU
+    else [(1,), (1024, 1024), (20, 320, 15), (16, 128, 64, 1280), (16, 7, 57, 32, 29)]
+)
 DISTRIBUTION_SHAPES = [(20, 320, 15)]
-REDUCTION_SHAPES = [(4096, 256 * i) for i in range(1, 10, 2)]
-MNK_SHAPES = [15, 160, 1024]
-REDUCTION_MNK_SHAPES = [(15, 160, 1024), (16, 1025, 255)]
-ONE_DIM_SHAPES = [(256 * i + 7,) for i in range(1, 10, 2)]
-
-DIM_POINTWISE_SHAPES = [
-    (1024, 1024, 1),
-    (16, 1024, 256),
-    (16, 7, 128, 64, 64),
-    (20, 320, 15),
-]
-DIMS = [[0], [-2], [2], [0, 2], [2, 1], [0, -1, 1]]
-TILE_DIMS = [(2,), (2, 0), (0, 2), (2, 2), (2, 2, 2), (2, 2, 2, 2)]
-REPEAT_SIZES = [(2, 3, 4, 5), (2, 0, 4, 5)]
+REDUCTION_SHAPES = [(2, 32)] if TO_CPU else [(1, 2), (4096, 256), (200, 40999, 3)]
+REDUCTION_SMALL_SHAPES = [(1, 32)] if TO_CPU else [(1, 2), (4096, 256), (200, 2560, 3)]
+NONZERO_SHAPES = [(2, 32)] if TO_CPU else REDUCTION_SHAPES + [(2637,)]
```

Comment:
There are many `OP_SHAPES` we defined here but only used once for the certain op, how about move them close to the file/lines where we use them

Reply:
> There are many `OP_SHAPES` we defined here but only used once for the certain op, how about move them close to the file/lines where we use them

Done.